certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -193,7 +193,7 @@ func (p *DeployableByOlmCheck) installedCSV(operatorData OperatorData) (string,
 
 	// query API server for the installed CSV field of the created subscription
 	go func(parent context.Context) {
-		ctx, cancel := context.WithTimeout(parent, subscriptionTimeout)
+		ctx, cancel := context.WithTimeout(parent, subscriptionTimeout*time.Second)
```

Comment:
Can we replace this in the assignment instead of inline? E.g.

```Go
var subscriptionTimeout = 180 * time.Second
```
And the same for other places? Reads a bit better to me, keeping the value and unit together.